src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+package com.denizenscript.depenizen.bukkit.commands.quests;
+
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+
+import me.blackvein.quests.Quest;
+import me.blackvein.quests.Quester;
+import me.blackvein.quests.Quests;
+import me.blackvein.quests.exceptions.InvalidStageException;
+import me.blackvein.quests.util.Lang;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+
+import org.bukkit.entity.Player;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.depenizen.bukkit.bridges.QuestsBridge;
+
+/**
+ * Command integration for Quests
+ * @author PikaMug
+ * @since 2.0.0-b595
+ */
+public class QuestsCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name Quests
+    // @Syntax quests [add/remove/set] (questId:<quest-id>) (stage-no:<#>) (points:<#>) (state:true/false)
+    // @Group Depenizen
+    // @Plugin Depenizen, Quests
+    // @Required 1
+    // @Short Edits quest player information.
+    //
+    // @Description
+    // This command allows you to give, quit, or set the stage of a quest for a player, or to add, subtract, or set
+    // the amount of Quest Points that a player holds.
+    //
+    // @Tags
+    // None
+    //
+    // @Usage
+    // Use to force player to take MyFirstQuest quest, ignoring requirements.
+    // - quests add quest-id:custom1 state:true

```

Comment:
`MyFirstQuest` and `custom1` don't line up. Unless these two values are in some way inherently linked by the Quests plugin for all servers, at least one of the values should be changed (eg, `Use to force the linked player to take the quest with id "custom1", ignoring requirements.`)

Reply:
`MyFirstQuest` would be a remnant from when I tested using quest names. Worked fine until I realized names with spaces in them like `My First Quest` (quite common in practice) wouldn't fly. I'll update the usage accordingly.